### PR TITLE
Update minor versions of Scala / Akka / Spray / spray-json

### PR DESCRIPTION
**Pull request by ajantis**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,7 +4,7 @@ name := "riak-scala-client"
 
 version := "0.9.5"
 
-scalaVersion := "2.11.5"
+scalaVersion := "2.11.7"
 
 crossScalaVersions := Seq("2.11.5", "2.10.4")
```
Comment (agemooij) [inline build.sbt]: Scala versions have gone up too (2.11.7 and 2.10.5 AFAIK). Might as well update those.

**Comment by agemooij** [comment]
Looks good for the rest.

File: build.sbt (hunk 1 above)
Comment (ajantis) [inline build.sbt]: ah, yes, good catch. I forgot these ones.

**Comment by ajantis** [comment]
Cool. Merged. Thanks!

**Pull request merged**